app/web/features/profile/view/leaveReference/LeaveReferencePage.tsx

```diff
@@ -76,15 +78,15 @@ export default function LeaveReferencePage() {
           user.friends === User.FriendshipStatus.FRIENDS) ||
         (hostRequestId &&
           availableRefrences.availableWriteReferencesList.find(
```

Comment:
```suggestion
          availableReferences.availableWriteReferencesList.find(
```
Ever since I installed the spell checker plugin, this typo has been annoying me :p so I think we should fix it here